src/ros/tf/tf_mongo.pl

```diff
@@ -226,11 +226,11 @@
 
 tf_mng_tree(Stamp, Tree) :-
 	% read all required transforms
-	lookup_transforms(Stamp, Transforms),
+	tf_mng_lookup_all(Transforms, Stamp),
```

Comment:
I tested tf_mng_tree with your changes and it works. Therefore I would merge atleast this part.

Reply:
Sounds good!